README.md

```diff
@@ -270,6 +270,51 @@ module.exports = {
 };
 ```
 
+If the tag name is not specified or assigned to the wild card `'*'` it will process all the tags.
+
+> You can use your custom filter to specify html elements to be processed.
+
+For example:
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.html$/i,
+        loader: 'html-loader',
+        options: {
+          attributes: {
+            list: [
+              {
+                // Tag name
+                tag: '*', // or tag: ''
```

Comment:
No need to support `tag: '*'`, if you don't provide `tag` try to use on all attributes, i.e `tag` can be `string` or `undefined` (not specify)

Reply:
It will work now when the string is empty or it's undefined